src/citrine/jobs/job.py

```diff
@@ -50,13 +77,37 @@ class JobStatusResponse(Resource['JobStatusResponse']):
 
     job_type = properties.String("job_type")
     """:str: the type of job for this status report"""
-    status = properties.String("status")
+    _status = properties.String("status")
     """:str: The status of the job. One of "Running", "Success", or "Failure"."""
     tasks = properties.List(Object(TaskNode), "tasks")
     """:List[TaskNode]: all of the constituent task required to complete this job"""
     output = properties.Optional(properties.Mapping(String, String), 'output')
     """:Optional[dict[str, str]]: job output properties and results"""
 
+    @property
+    def status(self) -> Union[JobStatus, str]:
```

Comment:
Since `BaseEnumeration` has `str` as a superclass, returning a `JobStatus` would not lead to a change in behavior, so this is not an API break.